backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/GwtMusic.java

```diff
@@ -74,7 +74,7 @@ public void stop () {
 
 	@Override
 	public boolean isPlaying () {
-		isPlaying &= sound.getPlayState() == 1;
+		isPlaying = sound.getPaused() == false & sound.getPlayState() == 1;
```

Comment:
I'd say that `&&` should be used instead.
Also, `sound.getPaused() == false` is just `!sound.getPaused()`